packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -336,6 +336,14 @@ def test_metrics_containers_nan(dcos_api_session):
         assert metric_value == '', 'unexpected metric value: {}'.format(metric_value)
 
 
+@retrying.retry(wait_fixed=(2 * 1000), stop_max_delay=(150 * 1000))
```

Comment:
Why retry this test?

Reply:
Thanks @adamtheturtle! I did not intend to retry the test. The `GET` request should be retried because it may take up to 20 seconds for the metrics to appear in Telegraf.